src/index.html

```diff
@@ -11,7 +11,157 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="form"
+      method="post"
+    >
+      <fieldset class="form__section">
+        <legend>Personal information</legend>
+        <div class="form__item">
+          <label>
+            Surname:
+            <input type="text" id="userName">
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            Name:
+            <input type="text" id="userSurname">
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              id="userAge"
+              value="12"
+              max="100"
+              min="1"
+            >
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            Full date of birth:
+            <input type="date" id="userBirth">
+          </label>
+        </div>
+        <div class="form__item">
+          <label>
+            I accept the term of the agreement
+            <input type="checkbox" id="userAgreement">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="form__section">
```

Comment:
```suggestion
      </fieldset>
      
      <fieldset class="form__section">
```
add spaces between blocks of code